nix/python/default.nix

```diff
@@ -0,0 +1,20 @@
+{ system, python3, pyperscan }:
+
+let
+  self = python3.override {
+    inherit self;
+    packageOverrides = final: prev: {
+      arpy = final.callPackage ../arpy { };
+      cstruct = final.callPackage ../cstruct { };
+      jefferson = final.callPackage ../jefferson { };
+      lief = prev.lief.overrideAttrs (super: {
+        meta.platform = super.meta.platform ++ [ final.lib.platforms.darwin ];
```

Comment:
Is this line still required with https://github.com/NixOS/nixpkgs/pull/217727 ?

Reply:
It needs to propagate to `nixos-unstable` branch first. It will take up to a few days, depending on when the next merge will happen.